mlflow/server/js/package.json

```diff
@@ -14,6 +14,7 @@
     "immutable": "3.8.1",
     "jquery": "3.0.0",
     "json-bigint": "databricks/json-bigint#a1defaf9cd8dd749f0fd4d5f83a22cd846789658",
+    "lodash": "^4.17.11",
```

Comment:
Including [lodash](https://lodash.com/docs/4.17.11) (a small dependency at 24 Kb gzipped) for some of the useful utility methods it provides.